fdbcli/fdbcli.actor.cpp

```diff
@@ -2670,6 +2661,20 @@ Reference<ReadYourWritesTransaction> getTransaction(Database db,
 	return tr;
 }
 
+// TODO: Update this function to get rid of Database and ReadYourWritesTransaction after refactoring
+Reference<ITransaction> getTransaction(Database db,
+                                       Reference<ReadYourWritesTransaction>& tr,
+                                       Reference<ITransaction>& tr2,
```

Comment:
Reasoning about the lifetimes of tr and tr2 is a bit weird. Could you add some explanatory comments or rework this?

Reply:
Thanks. I added comments for the function to avoid confusion.

Reply:
Done. Added comments for the lifetime and usage for the function.